bliss.shy.js

```diff
@@ -514,11 +512,16 @@ $.Element.prototype = {
 					transitionProperty: Object.keys(props).join(", ")
 				});
```

Comment:
Why set this if the duration is 0? Not to mention that it never gets undone since you only set `$.style(this, previous)` if the duration is not 0.